lib/Db/MessageMapper.php

```diff
@@ -487,9 +487,17 @@ public function deleteAll(Mailbox $mailbox): void {
 			->from($this->getTableName())
 			->where($messageIdQuery->expr()->eq('mailbox_id', $deleteRecipientsQuery->createNamedParameter($mailbox->getId())));
```

Comment:
```suggestion
			->where($messageIdQuery->expr()->eq('mailbox_id', $messageIdQuery->createNamedParameter($mailbox->getId())));
```

I'm pretty sure the parameter won't work otherwise. You always have to use the one that you use to execute the query.